CHANGELOG/README.md

```diff
@@ -0,0 +1,29 @@
+# CHANGELOGs
+
+## Development release
+
+- [CHANGELOG-1.18.md](./CHANGELOG-1.18.md)
+
+## Current release
+
+- [CHANGELOG-1.17.md](./CHANGELOG-1.17.md)
+
+## Older releases
```

Comment:
Do we really need that file? My concern is that it is hard to automate via `krel` to update that file since we have those 3 sections. We could also stick to a plain list here…

Reply:
@saschagrunert -- I'd like to keep the file as it's likely referenced in a few places outside of our repo. Agreed that it'll be easier to manage if we do a single list though and I've made that change in e54bfa4652d2f87c7c625bd2758a96a1fb8c4c7c.

Reply:
Sounds good, thanks :pray: